.eslintrc.cjs

```diff
@@ -12,12 +12,7 @@ const config = {
     ecmaVersion: "latest",
     sourceType: "module",
     tsconfigRootDir: __dirname,
-    project: [
-      "./tsconfig.json",
-      "./cli/tsconfig.eslint.json", // separate eslint config for the CLI since we want to lint and typecheck differently due to template files
-      "./upgrade/tsconfig.json",
-      "./www/tsconfig.json",
-    ],
+    projectService: true,
```

Comment:
https://typescript-eslint.io/blog/announcing-typescript-eslint-v8-beta#project-service: this is our snazzy new way of enabling typed linting. It should be faster and require less configuration.

I couldn't tell from quickly reading the code: what is the `tsconfig.eslint.json` doing differently? Is it still necessary?

---

Btw, I ran [hyperfine](https://github.com/sharkdp/hyperfine) measurements, and they showed promising performance improvements on my M1 Max Mac Studio:

* ~10% faster bumping to typescript-eslint from v7 to v8
* ~20% faster again switching from `project` to `projectService`

```plaintext
Benchmark 1: npx turbo lint --no-cache
  Time (mean ± σ):      7.464 s ±  0.094 s    [User: 32.604 s, System: 2.431 s]
  Range (min … max):    7.346 s …  7.665 s    10 runs
```

```plaintext
# baseline: typescript-eslint@v7 with project
Benchmark 1: npx turbo lint --no-cache
  Time (mean ± σ):      7.464 s ±  0.094 s    [User: 32.604 s, System: 2.431 s]
  Range (min … max):    7.346 s …  7.665 s    10 runs
```
```plaintext
# typescript-eslint@v8 with project
Benchmark 1: npx turbo lint --no-cache
  Time (mean ± σ):      6.662 s ±  0.235 s    [User: 29.944 s, System: 2.253 s]
  Range (min … max):    6.501 s …  7.254 s    10 runs
```

```plaintext
# typescript-eslint@v8 with projectService
Benchmark 1: npx turbo lint --no-cache
  Time (mean ± σ):      5.344 s ±  0.126 s    [User: 25.637 s, System: 1.683 s]
  Range (min … max):    5.204 s …  5.554 s    10 runs
```

...for an overall performance improvement of roughly **28%**.

Reply:
> I couldn't tell from quickly reading the code: what is the tsconfig.eslint.json doing differently? Is it still necessary?

I can't remember why it's there but from guessing it sounds like `projectService` does it for us :)